CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ option(TRITON_BUILD_PYTHON_MODULE "Build Python Triton bindings" OFF)
 #  used conditionally in this file and by lit tests
 find_package(Python3 REQUIRED COMPONENTS Development Interpreter)
 
+# Customized release build type with assertions
+set(CMAKE_C_FLAGS_RELWITHDEBINFOANDASSERTIONS "-O2 -g")
```

Comment:
Is this used in cmake anywhere?

Reply:
If it is a customed one, the variable name seems too lone.

Reply:
For `BuildType`, CMake will use `CMAKE_{C, CXX}_FLAGS_{BuildType}` as compilation flags.